xof/blake2xs/blake.go

```diff
@@ -61,6 +66,11 @@ func (x *xof) Reseed() {
 	x.impl = y.(*xof).impl
 }
 
+func (x *xof) Reset() {
+	y := New(x.seed)
```

Comment:
Same as above. Use Reset in New, not the other way around.

Reply:
Should be fixed now !